samples/matter/light_bulb/boards/nrf21540dk_nrf52840.conf

```diff
@@ -33,5 +20,6 @@ CONFIG_CHIP_OTA_REQUESTOR=n
 # Disable QSPI NOR
 CONFIG_CHIP_QSPI_NOR=n
 
-# Reduce application size
-CONFIG_USE_SEGGER_RTT=n
+# Disable Factory data feature since the board doesn't have external flash memory.
```

Comment:
Factory data is not part of the external flash memory so the comment needs another justification or be removed :)

Reply:
That's right. I wonder if we should have the overlay in dts to add a factory data partition (since we don't use Partition Manager for that target). @markaj-nordic @LuDuda @Damian-Nordic what do you think?

Reply:
I think we shouldn't change it before the release. I only think that no comment is better than a misleading one :)
Or you could write something like "Disable Factory data feature since this requires "factory_data" partition in the flash memory and the partition manager is not used when this sample is build for nRF21540 DK".